### PR TITLE
Add new POI category icons

### PR DESCRIPTION
**Pull request by MizukiTemma**
### Short description

This PR adds two new POI category icons (mental health and shelter)

### Proposed changes

- Add constants and svg files for the new icons
- <s>(Still in clearing: adjust API response, see the issue page)</s> Add a new API response (In case the current ´icon´ and ´color´ are not used in future, we can remove them)

### Side effects

- What should be the german translation of "shelter"? Is it "Shelter" in German too? 😁 

### Resolved issues

Fixes: #2698 

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

**Comment by lunars97** [comment]
I think Unterkunft is a suitable word for shelter :)

**Comment by JoeyStk** [comment]
> What should be the german translation of "shelter"? Is it "Shelter" in German too? 😁

I think "Unterkunft" certainly is an option, but in this case I'd probably prefer "Obdach" as it seems to have an immediate connection to the "Vernetztes Obdach" project. But maybe @f1sh1918 can give us some additional context on what the best translation for "shelter" would be :)

**Comment by FritjofK** [comment]
@MizukiTemma and @JoeyStk 
Please use "**Notunterkünfte**" as the German translation for shelter. That is the exact wording that is desired for "VernetztesObdach".

**Review by JoeyStk — APPROVED** [review]
LGTM! I think in the near future we should think about whether we want our categories to be in plural or singular terms, but for now it's consistent with what we have :)

File: integreat_cms/release_notes/current/unreleased/2698.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+en: Add mental health and shelter as new POI category icons
+de: Füge neue POI-Kategoriesymbole hinzu
```
Comment (seluianova) [inline integreat_cms/release_notes/current/unreleased/2698.yml:2]:
```suggestion
de: Füge Psychische Gesundheit und Notunterkünfte als neue POI-Kategoriesymbole hinzu
```

**Review by seluianova — APPROVED** [review]
LGTM, thanks!

**Pull request merged**